### PR TITLE
[FEATURE] Install guides-graphs for plantUML support

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +25,8 @@
     },
     "config": {
         "allow-plugins": {
-            "ergebnis/composer-normalize": true
+            "ergebnis/composer-normalize": true,
+            "phpstan/extension-installer": true
```
Comment (brotkrueml) [inline composer.json:29]: This one is not part of the PR?

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (linawolf) [reply composer.json]: removed it

**Review by brotkrueml — APPROVED** [review]
